scripts/tools/publish-nightly.js

```diff
@@ -51,6 +51,11 @@ const publishPaths = {
   testing_utilities: ['client-frameworks-support', 'testing-utilities', 'dist']
 };
 
+if (process.env.NIGHTLY === 'true' && !process.env.NIGHTLY_VERSION) {
```

Comment:
```suggestion
// Add container folder to publish in the nightly. Skip it for rc-release since container rc is done separately. 
if (!process.env.NIGHTLY_VERSION) {
  logHeadline('Added container to nightly release');
```
I think this should work better.